qualification/tied_array_channelised_voltage/test_linearity.py

```diff
@@ -70,12 +70,12 @@ async def _test_linearity(
     assert scales[middle] == pytest.approx(1.0)
     powers = np.zeros_like(scales)
     for batch in batched(enumerate(scales), len(receiver.stream_names)):
-        for (_, scale), beam_name in zip(batch, receiver.stream_names):
+        for (_, scale), beam_name in zip(batch, receiver.stream_names, strict=False):
```

Comment:
I think a comment like you've done in `acv/__init__.py` would be helpful here.

Reply:
Done.